### ISSUE TITLE
dcat:Catalog: publisher granularity

### ISSUE THREAD

**Issue by huberrob** [SEMICeu/DCAT-AP#386]
In DCAT-AP class dcat:Catalog has a dc:publisher property with granularity 1. 
However there are some repositories/catalogs which are maintained by several organisations which share responsibilities such as PANGAEA which is jointly operated by awi.de and marum.de. 
Therefore, changing granularity of d:publisher in this class should be changed to 1...*

**Comment by bertvannuffelen** [comment]
@huberrob thanks for this feedback.

To better understand your request some followup questions:

A publisher is the organisation that takes the responsability to provide access to the catalogue. Ownership (or responsible for the content) is often the same organisation, but not always. Is that a distinction you want to make?  
In your case is the catalogue being made available by multiple organisations on different (web)locations?  
If someone has problems with the catalogue which organisation is than the first contact point or are both to be contacted?

**Comment by huberrob** [comment]
No, I am specifically referring to the role **publisher** (DCTERMS: 'An entity responsible for making the resource available').  For example PANGAEA is operated by two organisations which equally share this responsibility (tech teams and content teams on both sides) so there are two organisations which **both** act as publisher (and are very picky with this;).

**Comment by init-dcat-ap-de** [comment]
The range is a `foaf:Agent`. If you need more than one publisher, you can always use the [`foaf:Group`](http://xmlns.com/foaf/spec/#term_Group) class to bundle all members of the publisher group. 

```
<ds123> a dcat:Dataset ;
  dcterms:publisher [
    a foaf:Group, foaf:Agent ;
    foaf:name "PANGEA Publisher Group" ;
    foaf:member [
      a foaf:Agent ;
      foaf:name "Org 1" ;
    ] , [
      a foaf:Agent ;
      foaf:name "Org 2" ;
    ] ;
  ] .
```

You could also use the [Organization Ontology](https://www.w3.org/TR/vocab-org/#org:Organization).

**Comment by bertvannuffelen** [comment]
According to the feedback during the webinar the community recommends to follow the above suggested approach to use foaf:Group.

**Comment by bertvannuffelen** [comment]
I added this resolution to the draft release at https://semiceu.github.io/DCAT-AP/releases/3.0.1-draft/#Catalogue.publisher
